### ISSUE TITLE
Review instrumentation of `colorProfile`

### ISSUE THREAD

**Issue by arouinfar** [phetsims/density-buoyancy-common#261]
In https://github.com/phetsims/buoyancy/issues/51#issuecomment-1979357490 the devs asked:

> All the colors listed in Buoyancy and Density are phetioReadOnly:true. Why is this valuable to have? Should they be uninstrumented?

The relevant paper trail is in https://github.com/phetsims/density-buoyancy-common/issues/65 and it appears that the instrumented ColorProperties may be required for State.

I'd like to review `global.view.colors` for all sims in the suite and decide: 
* Should any additional colors be instrumented?
* Are there any ColorProperties that would benefit from being `phetioReadOnly: false`?

**Comment by arouinfar** [comment]
The mystery fluid colors still use the old "Density A", etc. naming scheme and should likely be updated to the new "Fluid A" language introduced in https://github.com/phetsims/density-buoyancy-common/issues/248

<img width="310" alt="image" src="https://github.com/user-attachments/assets/a13c0f93-400f-4498-80d6-a26bff71ede2">

Edit: I reported this in #265.

**Comment by arouinfar** [comment]
Here is the full list of the instrumented colors. All 3 density-buoyancy-common sims are the same, even though these colors don't necessarily appear in all sims. In general, these colors correspond to the fluids in the pool and the materials inside the bottle, which seem like useful things to let clients control.

<img width="318" alt="image" src="https://github.com/user-attachments/assets/fe3848ac-119a-4509-9b7c-2e941a16ff27">

I'd like to wait for #265 to wrap up before working on this issue.

**Comment by arouinfar** [comment]
The design requests in #265 have wrapped up, but I think it makes the most sense to review this along with the materials over in https://github.com/phetsims/density-buoyancy-common/issues/250.

**Comment by arouinfar** [comment]
We discussed this in today's design meeting over in https://github.com/phetsims/density-buoyancy-common/issues/268#issuecomment-2253135662.

I'll leave this on hold until #268 is completed.

**Comment by AgustinVallejo** [comment]
#268 Is almost done, and no longer holding this one.

**Comment by arouinfar** [comment]
Density does not require any instrumentation of the color profile, as the Mystery blocks all have a `colorProperty` as a part of the `customMaterial`.

Buoyancy and Buoyancy: Basics both have the same set of instrumented colors:
<img width="300" alt="image" src="https://github.com/user-attachments/assets/5d8d819a-7ae9-44d9-9ee9-9872f88398d6">

None of these colors need to be customizable, so I recommend uninstrumenting them. In https://github.com/phetsims/density-buoyancy-common/issues/268#issuecomment-2253135662, we noted that customizing fluid colors, in particular, could be problematic.

It would be nice to customize the colors of the mystery materials (T-Y) that don't have a textured appearance. Can we instrument those colors?

To summarize:
* No changes needed for Density
- [x] For  Buoyancy & Buoyancy: Basics: Uninstrument all currently instrumented colors.
- [x] For Buoyancy only: Instrument the colors of mystery materials T-Y

**Comment by samreid** [comment]
In the commits, I uninstrumented all ColorProperties, then added read/write instrumentation for the mysteries T-Y. In studio I was able to customize the color, and I saw it was preserved in Test. Please review.

**Comment by arouinfar** [comment]
Looks good on main, thanks @samreid! Closing.
